app/models/patron.rb

```diff
@@ -48,17 +53,17 @@ def stale?
     record == { 'messageList' => [{ 'code' => 'sessionTimedOut', 'message' => 'The session has timed out.' }] }
   end
 
-  def standing
-    fields.dig('standing', 'key')
-  end
-
-  def standing_alert?
-    PATRON_STANDING_ALERTS.include?(standing)
+  def standing_alert
+    PATRON_STANDING_ALERTS[standing] || ''
   end
 
   private
 
     def fields
       record['fields']
     end
+
+    def standing
```

Comment:
:nods:

Reply:
there is a pattern of doing `thing_code` and `thing_human` ... like item type in bib record and pickup library in hold and status in fines. Would that make sense here too?